string team_name
string uav_name
RPY rpy
pose pose